### PR TITLE
Remove the release handling for Katello

### PR DESCRIPTION
**Pull request by ekohl**
The release field is no more. It has ceased to be.

### CODE REVIEW

**Comment by jlsherrill** [comment]
I think this makes sense.  We as katello need to adjust our policy to use the triaged flag, but that involves updating all our existing issues to use the backlog.  This might require some custom scripting as there is currently 1000 of them. @GregSutcliffe is there a simple way to set the triaged flag on any open issue currently with a target release of 'Katello Backlog'?

**Comment by GregSutcliffe** [comment]
@jlsherrill I can do it from the console if you want, which should also skip emailling people about that change. Ping me on IRC when you have a moment and we'll get it sorted

File: app.rb [hunk 1]
```diff
@@ -63,10 +63,6 @@
           pull_request.add_comment(message)
         end
       elsif !issue.rejected?
-        if project.name == 'Katello' && issue.release == 'Katello Backlog'
-          issue.set_release(nil)
-        end
```
Comment (jlsherrill) [inline app.rb]: @ekohl okay, we've moved our process to use the triaged flag, can you update this to set triaged to 'false' instead of this?

Comment (ekohl) [reply app.rb]: Should this still be limited to Katello or does Foreman follow a similar pattern?

Comment (jlsherrill) [reply app.rb]: I'm not sure, @tbrisker @GregSutcliffe have you guys moved to use the triaged flag yet?  and if so would you like for issues to 'untriage' when a pr gets opened?

Comment (ekohl) [reply app.rb]: Having thought about it, we can just make a boolean in the config if this behavior is desired. That makes it trivial to enable it for other repos too.

**Comment by ekohl** [comment]
I'm going to merge this now but we should implement code to set triaging to false after merging and perhaps set the `Fixed in` field to include the matching version.

**Pull request merged**